scripts/qa_crowbarsetup.sh

```diff
@@ -285,6 +285,13 @@ function addcloudpool
         "cloudpool"
 }
 
+function add_salt_repo
+{
+    # TODO: use $reposerver_url
+    zypper ar http://$susedownload/ibs/SUSE/Products/SLE-Module-Adv-Systems-Management/12/x86_64/product/ salt
+    zypper ar http://$susedownload/ibs/SUSE/Updates/SLE-Module-Adv-Systems-Management/12/x86_64/update/ saltup
```

Comment:
update folder does not exist

Reply:
Perhaps you need to refresh it also?
Something like: zypper -n --gpg-auto-import-keys ref

Reply:
The repo is signed by the default trusted SLE key and it will be auto-refreshed on first use - but why do you say 'update folder does not exist' ?